docs/sources/configure/examples/15-Memberlist-Ring-Snippet.yaml

```diff
@@ -0,0 +1,15 @@
+# doc-example:skip-validation=true
+# This partial configuration uses memberlist for the ring.
+
+common:
+  ring:
+    instance_addr: 127.0.0.1
```

Comment:
should this be dropped when loki is running as a cluster? otherwise we'd not be able to distinguish the members.

Reply:
Good catch! Removed.